### DISCUSSION TITLE
Toku

### DISCUSSION THREAD

**Discussion by epicmemer6** [iptv-org/community#706] [Q&A]
**Channel Name:** Toku
**Channel ID:** Toku.us
**Country:** United States
**Language:** English, Spanish (SAP)
**Website:** tokuhd.com

**Comment by Sphinxroot** [comment]
some are provided in pluto usa 

#EXTINF:-1 tvg-id="PlutoTVTokuSHOUTsu.us" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5c3f8f12a93c2d61b9990a4e/colorLogoPNG.png" group-title="Series",Pluto TV TokuSHOUTsu (720p)
http://service-stitcher.clusters.pluto.tv/stitch/hls/channel/5c3f8f12a93c2d61b9990a4e/master.m3u8?advertisingId=&appName=web&appStoreUrl=&appVersion=DNT&app_name=&architecture=&buildVersion=&deviceDNT=0&deviceId=5c3f8f12a93c2d61b9990a4e&deviceLat=&deviceLon=&deviceMake=web&deviceModel=web&deviceType=web&deviceVersion=DNT&includeExtendedEvents=false&marketingRegion=US&serverSideAds=false&sid=848&terminate=false&userId=
#EXTINF:-1 tvg-id="PlutoTVTokuSHOUTsu.us" tvg-country="US" tvg-language="English" tvg-logo="http://images.pluto.tv/channels/5c3f8f12a93c2d61b9990a4e/colorLogoPNG.png" group-title="Series",Pluto TV TokuSHOUTsu (720p)
https://service-stitcher.clusters.pluto.tv/v1/stitch/embed/hls/channel/5c3f8f12a93c2d61b9990a4e/master.m3u8?advertisingId=channel&appName=rokuchannel&appVersion=1.0&bmodel=bm1&channel_id=channel&content=channel&content_rating=ROKU_ADS_CONTENT_RATING&content_type=livefeed&coppa=false&deviceDNT=1&deviceId=channel&deviceMake=rokuChannel&deviceModel=web&deviceType=rokuChannel&deviceVersion=1.0&embedPartner=rokuChannel&genre=ROKU_ADS_CONTENT_GENRE&is_lat=1&platform=web&rdid=channel&studio_id=viacom&tags=ROKU_CONTENT_TAGS


other than that its a paid sub https://www.watchtoku.com/

**Comment by AntiPontifex** [comment]
Already in list: please close.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19711
